code/update.lua

```diff
@@ -1,7 +1,9 @@
 print("Installing cc-code...")
 
-fs.delete("code.lua")
-fs.delete("code")
+if fs.exists("code/") and fs.exists("code.lua") then
```

Comment:
```suggestion
if fs.exists("/code") and fs.exists("/code.lua") then
```

The slash in the `fs.exists("code/")` doesn't really do anything. (It still returns `true` if `code` was a file instead of a directory.)

Prefixing with `/` allows running update when in a different directory. It has to be installed in the root folder always anyway because of the limitations that CC brings with itself. (I really wish they took some inspiration from OpenComputers here...)